setup

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+ID=$(id -ur)
+if [ $ID != 0 ]; then
+    echo "This script should be run as root (when used in console mode), use sudo." >&2
+    exit 1
+fi
+
+UPDATEVM=`qubes-prefs --force-root updatevm`
+
+if [ -z "$UPDATEVM" ]; then
+    echo "UpdateVM not set, exiting"
+    exit 1
+fi
+
+# Create directories and copy files
+mkdir -p /usr/share/fwupd-qubes/
+cp -R src/ /usr/share/fwupd-qubes/
+cp -R test/ /usr/share/fwupd-qubes/
+qvm-run --nogui -q -u root $UPDATEVM 'mkdir /usr/lib/qubes-fwupd' || exit 1
+
+# Create symlink
+rm /bin/qubes-fwupdmgr
+ln -s /usr/share/fwupd-qubes/src/qubes_fwupdmgr.py /bin/qubes-fwupdmgr
```

Comment:
In a final solution we will use [qubes-builder](https://github.com/QubesOS/qubes-builder/) to install the package (it will be added in further development). This script is for only development proposes.